.github/workflows/release.yml

```diff
@@ -20,77 +17,79 @@ jobs:
+        run: npm test
+        env:
+          CI: true
 
-    # Build electron app package installers
-    - name: package electron - windows
-      if: startsWith(matrix.os, 'windows')
-      run: npm run package:windows
-    - name: package electron - linux
-      if: startsWith(matrix.os, 'ubuntu')
-      run: npm run package:linux
-    - name: package electron - mac
-      if: startsWith(matrix.os, 'mac')
-      run: npm run package:mac
-    - name: npm rebuild - mac
-      if: startsWith(matrix.os, 'mac')
-      run: npm rebuild
-    - name: Mac installer
-      if: startsWith(matrix.os, 'mac')
-      run: npm run installer:mac
+      # Build electron app package installers
+      - name: package electron - windows
+        if: startsWith(matrix.os, 'windows')
+        run: npm run package:windows
+      - name: package electron - linux
+        if: startsWith(matrix.os, 'ubuntu')
+        run: npm run package:linux
+      - name: package electron - mac
+        if: startsWith(matrix.os, 'mac')
+        run: npm run package:mac
+      - name: npm rebuild - mac
+        if: startsWith(matrix.os, 'mac')
+        run: npm rebuild
+      - name: Mac installer
+        if: startsWith(matrix.os, 'mac')
+        run: npm run installer:mac
 
-    # Upload installers to github release
+      # Upload installers to github release
 
-    - name: Upload app to release - windows
-      if: startsWith(matrix.os, 'windows')
-      uses: svenstaro/upload-release-action@v2
-      with:
-        repo_token: ${{ secrets.GITHUB_TOKEN }}
-        file: dist/installers/${{ steps.package_info.outputs.package_name }}-${{ steps.package_info.outputs.package_version }}-setup.exe
-        asset_name: ${{ steps.package_info.outputs.package_name }}-${{ steps.package_info.outputs.package_version }}-${{ matrix.setting }}-setup.exe
-        tag: ${{ github.ref }}
-    - name: Upload app to release - linux
-      if: startsWith(matrix.os, 'ubuntu')
-      uses: svenstaro/upload-release-action@v2
-      with:
-        repo_token: ${{ secrets.GITHUB_TOKEN }}
-        file: dist/installers/${{ steps.package_info.outputs.package_name }}_${{ steps.package_info.outputs.package_version }}_x64.deb
-        asset_name: ${{ steps.package_info.outputs.package_name }}_${{ steps.package_info.outputs.package_version }}_${{ matrix.setting }}_x64.deb
-        tag: ${{ github.ref }}
-    - name: Upload app to release - mac
-      if: startsWith(matrix.os, 'mac')
-      uses: svenstaro/upload-release-action@v2
-      with:
-        repo_token: ${{ secrets.GITHUB_TOKEN }}
-        file: dist/installers/${{ steps.package_info.outputs.package_name }}-${{ steps.package_info.outputs.package_version }}.dmg
-        asset_name: ${{ steps.package_info.outputs.package_name }}-${{ steps.package_info.outputs.package_version }}-${{ matrix.setting }}.dmg
-        tag: ${{ github.ref }}
+      - name: Upload app to release - windows
+        uses: svenstaro/upload-release-action@v2
+        with:
```

Comment:
all of the `if` statements seem to have been lost with this update, was that on purpose?